src/Sulu/Bundle/AdminBundle/phpunit.xml.dist

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<phpunit bootstrap="./Tests/bootstrap.php" colors="true">
+<phpunit bootstrap="../TestBundle/Resources/app/bootstrap.php" colors="true">
```

Comment:
If we were to make a subtree split then we would reference the testBundle from within the `vendor` directory.